src/components/molecules/NewsFlashFilterPanel.tsx

```diff
@@ -41,59 +41,65 @@ const useStyles = makeStyles({
   },
 });
 
-export const NewsFlashFilterPanel: React.FC = () => {
+interface IProps {}
```

Comment:
Why we have an empty interface?

Reply:
I declare that if you will need pass props to this component, use this interface